fs_test.go

```diff
@@ -44,6 +45,186 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+func TestSeeker(t *testing.T) {
+	s3cl := newClient(t)
+
+	const testFile = "file.txt"
+	content := []byte("content")
+
+	createBucket(t, s3cl, *bucket)
+	cleanBucket(t, s3cl, *bucket)
+
+	writeFile(t, s3cl, *bucket, testFile, content)
+
+	t.Cleanup(func() {
+		cleanBucket(t, s3cl, *bucket)
+
+		t.Log("test stats:")
+		t.Log("ListObjects calls:", atomic.LoadInt64(&listC))
+		t.Log("GetObject calls:", atomic.LoadInt64(&getC))
+	})
+
+	t.Run("seek once", func(t *testing.T) {
+		fixtures := []struct {
+			desc     string
+			offset   int64
+			whence   int
+			expected int64
+			err      error
+		}{
+			{"whence SeekStart ", 2, io.SeekStart, 2, nil},
```

Comment:
there are too many arguments and it's hard to follow, can we please do the following?

```go
			{
				desc:     "whence SeekStart ",
				offset:   2,
				whence:   io.SeekStart,
				expected: 2,
				err:      nil,
			},
```

Please apply the same for all other tests